src/AzurePipelinesToGitHubActionsConverter.Core/Conversion.cs

```diff
@@ -185,30 +181,30 @@ private ConversionResponse ConvertAzurePipelineToGitHubActionV2(string yaml, boo
 
                 //Strategy
                 JsonElement strategy = new JsonElement();
-                if (jsonObject.TryGetProperty("strategy", out jsonElement) == true)
+                if (jsonObject.TryGetProperty("strategy", out jsonElement) )
```

Comment:
would be nice to autoformat the changed files and remove the extra whitespace between final brackets

Reply:
I usually do - missed this one. Thank you.